### DISCUSSION TITLE
after using "Vertical Taskbar for Windows 11" my start menu is messed up

### DISCUSSION THREAD

**Discussion by PeedrParka** [ramensoftware/windhawk-mods#1389] [Q&A]
my start menu is fixed to the right of my screen, even after deactivating the mod
![Screenshot 2024-12-29 184520](https://github.com/user-attachments/assets/4cd57bfd-138a-4790-8817-799484da1206)
i need help because its very impractical

**Comment by m417z** [comment]
Answered here: https://github.com/ramensoftware/windhawk-mods/issues/1390#issuecomment-2564814671.
